tests/utils.py

```diff
@@ -97,7 +97,7 @@ def formatError(self, test, err):
 
 		_, module, call = test.address()
 
-		output = ['python2', 'run_tests.py', u'%s:%s' % (module, call)]
+		output = ['python2', 'run_tests.py', u'{0}:{1}'.format(module, call)]
```

Comment:
With Python 2.6 is working `'{}'.format('a')`, but not `'{}:{}'.format('a', 'b')`. You have to use `'{0}:{1}'.format('a', 'b')`. But better shoud be `'{module}:{call}'.format(module='a', call='b')`.